.gitattributes

```diff
@@ -1,39 +1,64 @@
+# Git LFS inclusion / exclusion is controlled by this file. Our approach is to:
+# INCLUDE dirs in Git LFS broadly, and then EXCLUDE text file-types narrowly.
+#
+# Why whitelist? Once a large binary file is committed to LFS, it is VERY
+# hard to remove it from the git history as it requires a history rewrite.
+# Once merged to `staging`, that NNN MB file makes our git repo that much
+# bigger forever".
+#
+# In contrast, if a text file is accidentally included in LFS, adding it back to
```

Comment:
Since text is broadly defined here in a way that may not be familiar to someone with Git experience, is it worth leaving a comment about what it includes? "In this context, 'text file' refers to any file that Git normally recognizes as text because it contains only printable characters and white space. It covers many file types, including source code, HTML, CSS, JSON, and YAML files, assuming they don't contain binary data."?

Reply:
Also, TBD if my Googling there is correct 😅

Reply:
Honestly: people without git experience shouldn't be in .gitattributes.

Reply:
Corollary to hot take: People without the experience to interpret "text file" shouldn't be reviewing the PR to .gitattributes 😂 🙈

Reply:
🔥 BURN 🔥